From 9c34685ebbdd6267e836805c236a791ac27da5de Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Wed, 10 Dec 2014 15:50:24 -0800
Subject: [PATCH 035/184] MIPS: OCTEON: Properly fix octeon_error_injector.

Source: Cavium Networks, Inc.
MR: 00000
Type: Integration
Disposition: Merged from Octeon Tree
ChangeID: 185fc09e65189cec7f0121a92821cb378f8bc8e5
Description:

Don't bring in the mess that is cvmx-fpa.c, but rather just directly
call cvmx_fpa1_enable().

Signed-off-by: David Daney <david.daney@cavium.com>
Signed-off-by: Corey Minyard <cminyard@mvista.com>
[Original patch taken from octeon-linux-kernel-patches-SDK-3.1.2-release]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/cavium-octeon/octeon-error-injector.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/cavium-octeon/octeon-error-injector.c b/arch/mips/cavium-octeon/octeon-error-injector.c
index 6b9c367..57c8b7f 100755
--- a/arch/mips/cavium-octeon/octeon-error-injector.c
+++ b/arch/mips/cavium-octeon/octeon-error-injector.c
@@ -60,7 +60,7 @@ static void octeon_error_injector_fpa1(void)
 	char *mem;
 	u64 old_threshold;
 
-	cvmx_fpa_enable();
+	cvmx_fpa1_enable();
 
 	mem = kmalloc(1024, GFP_KERNEL);
 	if (!mem) {
-- 
1.9.1

